zookeeper-server/src/main/java/org/apache/zookeeper/common/X509Util.java

```diff
@@ -159,6 +159,16 @@ public static ClientAuth fromPropertyValue(String prop) {
     private String sslClientAuthProperty = getConfigPrefix() + "clientAuth";
     private String sslHandshakeDetectionTimeoutMillisProperty = getConfigPrefix() + "handshakeDetectionTimeoutMillis";
 
+    /**
+     * Config properties for ZNode group ACL features
+     */
+    public static final String ZNODE_GROUP_ACL_CONFIG_PREFIX = "zookeeper.ssl.znodeGroupAcl.";
```

Comment:
This (and all znode group acl features) is an X509 feature - this is why the config belongs here. It's neither a X509Client or a X509Quorum config, so it was placed here. This config is on par with other `ssl*Properties` listed right above this block of code. Does this help?

Reply:
Maybe we can have a GroupAclUtil for all the stuff that we will add for ZK security. I can forsee that we will add multiple configurations, and it might be clearer to separate them with the current zk server code.

Reply:
If you believe this item belongs to the same ssl related config family, why not using 'getConfigPrefix() + "setX509ClientIdAsAcl"'?

And from my perspective, this is not SSL or X509 feature, it is an acl management feature. For example, if one day we have another similar requirement to set IP address as the ID (bad example, but just to show the relavance), then we have SET_IP_AS_ACL. This item, however, would be better to colocate with the SET_X509_CLIENT_ID_AS_ACL definition. And it should not be in this class.